armi/physics/fuelCycle/fuelHandlerInterface.py

```diff
@@ -36,6 +36,23 @@ class FuelHandlerInterface(interfaces.Interface):
         :id: I_ARMI_SHUFFLE
         :implements: R_ARMI_SHUFFLE
 
+        This interface allows for a user to define custom shuffle logic that
+        modifies to the core model. Being based on the :py:class:`~armi.interfaces.Interface`
+        class, it has direct access to the current core model.
```

Comment:
```suggestion
        This :py:class:`~armi.interfaces.Interface` allows a user to define custom
        shuffle logic that modifies to the core model.
```

Reply:
I just thought the two sentences could be combined into one.  I'm not married to it, and I won't die on this hill.